integreat_cms/cms/templates/pages/page_tree_node.html

```diff
@@ -4,8 +4,8 @@
 {% load tree_filters %}
 
 {% get_translation page language.slug as page_translation %}
-<tr id="page-{{ page.id }}-drop-left" data-drop-id="{{ page.id }}" data-drop-position="left" class="drop {% if page.relative_depth == 1 %} drop-between {% endif %} h-3 -m-3 hidden level-{{page.relative_depth}}"><td colspan="9"><div><span></span></div></td></tr>
-<tr id="page-{{ page.id }}" data-drop-id="{{ page.id }}" data-drop-position="last-child" class="drop drop-on border-t border-b border-solid border-gray-200 hover:bg-gray-100 {% if page.relative_depth > 1 %} level-{{page.relative_depth}} {% if not filter_form.has_changed %} hidden {% endif %} {% endif %}">
+<tr id="page-{{ page.id }}-drop-left" data-drop-id="{{ page.id }}" data-drop-position="left" class="drop drop-between h-3 -m-3 hidden level-{{page.relative_depth}}"><td colspan="9"><div><span></span></div></td></tr>
```

Comment:
Thank you very much for your review, I applied your suggestions so far and they work greatly.
Here I think we have a problem, committing this suggestion will break drag and drop, because the relative depth for children of roots is 2, so they will not render this additional drop region above the current page after expanding, I remove this if statement therefore.
But I can try to find an if statement, so that it will render for children and not for search results and archived pages